composer.json

```diff
@@ -21,9 +21,9 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0",
+        "php": ">=5.6",
         "gambitph/titan-framework": "1.9.*",
-        "ericmann/wp-session-manager": "1.2.*"
```

Comment:
I just saw that 1.2.2 was released last week for WP Session. I'd rather keep the required version as `1.2.*` so that we can keep getting updates when pulling it. If `composer.lock` was outdated, then we sure need to update it, but keeping 1.2.x as the requirement.